src/styles/reference.scss

```diff
@@ -0,0 +1,168 @@
+.reference-page {
+  table {
+    width: 100%;
+  }
+
+  h2,
+  h3 {
+    margin-top: 20px;
+    margin-bottom: 10px;
+  }
+}
+
+.api-modal-container {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  background-color: rgba(0, 0, 0, 0.5);
+  align-items: center;
+  justify-content: center;
+  z-index: 99999;
+
+  &--open {
+    display: flex;
+  }
+}
+
+.api-modal {
+  width: 800px;
+  max-width: 90vw;
+  max-height: 90vh;
+  border-radius: var(--amplify-radii-medium);
+}
+
+.api-model__header {
+  align-items: flex-start;
+}
+
+.api-modal__breadcrumbs {
+  flex: 1 0 0;
+  padding-bottom: var(--amplify-space-medium);
+  margin-bottom: var(--amplify-space-xs);
+  overflow: scroll;
+  align-items: baseline;
+  gap: 2px;
+  &__separator {
+    color: var(--amplify-colors-neutral-90);
+  }
+}
+
+.api-modal__breadcrumbs__current {
+  font-weight: bold;
+}
+
+.api-modal__content {
+  gap: var(--amplify-space-xxs);
+  grid-template-columns: 1fr;
+  grid-template-rows: auto;
+  align-items: baseline;
+
+  dt {
+    color: var(--amplify-colors-font-secondary);
+  }
+
+  dd {
+    padding-inline-start: var(--amplify-space-small);
+  }
+
+  &__name {
+    text-align: start;
+    color: var(--amplify-colors-neutral-90);
+    font-weight: bold;
+    align-items: baseline;
+  }
+
+  &__description,
+  &__value {
+    color: var(--amplify-colors-neutral-90);
+  }
+}
+
+.api-modal__api-name {
+  word-break: break-word;
+  flex: 1 0 0;
+}
+
+.api-modal__api-value {
+  overflow: scroll;
```

Comment:
I think if we set the overflow to `auto` it could hide the empty scrollbars from the element, but still allow for scrolling?